src/mobu/storage/nublado.py

```diff
@@ -802,7 +844,17 @@ def _extract_xsrf(self, response: Response) -> str | None:
         """
         cookies = Cookies()
         cookies.extract_cookies(response)
-        return cookies.get("_xsrf")
+        xsrf = cookies.get("_xsrf")
+        if xsrf is not None:
+            self._logger.debug(
+                "Extracted _xsrf cookie",
+                method=response.request.method,
+                url=response.url.copy_with(query=None, fragment=None),
+                status_code=response.status_code,
+                # Logging the set-cookie header can be useful here but it
+                # leaks secrets.  Don't put that code in a release build.
+            )
```

Comment:
Would we want to put this in each auth method so it would be easier to tell if it happened during the hub auth or the lab auth? It would be duplicated, but maybe more clear?

Reply:
Well, if we end up using it, we log that (with whether it is lab or hub as well).  I'm not sure we even want to log it in the extraction step.  Maybe I'll just leave the comment.

Reply:
Eh, I'm going to leave this alone for now.  If it's too chatty if we need to turn on debug for real, we can change it then.